### ISSUE TITLE
Cannot authenticate successfully and connect the clusters.

### ISSUE THREAD

**Issue by joydesigner** [kontena/lens#5473]
**Describe the bug**

E0527 16:26:25.414166 51680 proxy_server.go:147] Error while proxying request: Service Unavailable

Service Unavailable

**To Reproduce**
1. Launch the lens desktop for Mac
 (lastest version: Lens: 5.5.1-latest.20220526.1
Electron: 14.2.9
Chrome: 93.0.4577.82
Node: 14.17.0
2. Click on AKS cluster name (authenticated via Kubeconfig)

**Expected behavior**
Allow to connect to cluster

**Screenshots**

**Environment (please complete the following information):**
- Lens Version: Lens: 5.5.1-latest.20220526.1
- OS: OSX
- Installation method (Mac DMG Mac OS):

**Logs:**
When you run the application executable from command line you will see some logging output. Please paste them here:
```
Your logs go here...
E0527 16:26:25.414166 51680 proxy_server.go:147] Error while proxying request: Service Unavailable

Service Unavailable

**Kubeconfig:**

your kubeconfig here

Will share via private channel.

**Comment by df-cgdm** [comment]
same here (on Windows with version 5.5.1)
here is the error logs:
`error:   Ôû¬ [CLUSTER]: Failed to connect to "xxxxxx" : StatusCodeError: 404 - {"paths":["/apis","/apis/","/apis/apiextensions.k8s.io","/apis/apiextensions.k8s.io/v1","/apis/apiextensions.k8s.io/v1beta1","/healthz","/healthz/etcd","/healthz/log","/healthz/ping","/healthz/poststarthook/crd-informer-synced","/healthz/poststarthook/generic-apiserver-start-informers","/healthz/poststarthook/priority-and-fairness-config-consumer","/healthz/poststarthook/priority-and-fairness-filter","/healthz/poststarthook/start-apiextensions-controllers","/healthz/poststarthook/start-apiextensions-informers","/livez","/livez/etcd","/livez/log","/livez/ping","/livez/poststarthook/crd-informer-synced","/livez/poststarthook/generic-apiserver-start-informers","/livez/poststarthook/priority-and-fairness-config-consumer","/livez/poststarthook/priority-and-fairness-filter","/livez/poststarthook/start-apiextensions-controllers","/livez/poststarthook/start-apiextensions-informers","/metrics","/openapi/v2","/readyz","/readyz/etcd","/readyz/informer-sync","/readyz/log","/readyz/ping","/readyz/poststarthook/crd-informer-synced","/readyz/poststarthook/generic-apiserver-start-informers","/readyz/poststarthook/priority-and-fairness-config-consumer","/readyz/poststarthook/priority-and-fairness-filter","/readyz/poststarthook/start-apiextensions-controllers","/readyz/poststarthook/start-apiextensions-informers","/readyz/shutdown","/version"]} +519ms

**Comment by Nokel81** [comment]
Is this a duplicate of https://github.com/lensapp/lens/issues/5462?

**Comment by ncastrocosta** [comment]
5.5.2 and 5.5.1 didn't work for me as well. Back to 5.4.6 and everything is fine again. It was saying I had invalid credentials. Very simple log message, but not true.

**Comment by Nokel81** [comment]
What type of cluster are you trying to connect to? Maybe this is actually a duplicate of #5455

**Comment by jachro** [comment]
Hi. I'm getting the same problem as @ncastrocosta in exactly the same environment as described in #5455

**Comment by adamkbmc** [comment]
Same issue here, Lens 5.5.1 on Windows - I can connect to AWS EKS, GKE and even Oracle cloud clusters but local Rancher clusters all fail with the bogus "Invalid Credentials" error (the same kubeconfigs work from kubectl via the CLI from the same machines)

**Comment by Oceanswave** [comment]
Same issue as @adamkbmc connecting to a rancher cluster fails, but kubectl works fine - this stopped working in a recent release and works on 5.4.6

**Comment by Nokel81** [comment]
@Oceanswave can you run lens from the command line with `DEBUG=true` and share some logs?

**Comment by tgruenert** [comment]
some here as @Oceanswave mentioned. 

After setting DEBUG=true only some logs at startup will shown. 
```
$ ./Lens.exe
info:    ▪ 📟 Setting Lens as protocol client for lens:// +0ms
info:    ▪ 📟 Protocol client register succeeded ✅ +1ms
debug:   ▪ [APP-MAIN] configuring packages +0ms
info:    ▪ [EXTENSIONS-STORE]: LOADING from  ... +16ms
info:    ▪ [EXTENSIONS-STORE]: LOADED from C:\Users\tgruenert\AppData\Roaming\Lens\lens-extensions.json +9ms
info:    ▪ [FILESYSTEM-PROVISIONER-STORE]: LOADING from  ... +9ms
info:    ▪ [FILESYSTEM-PROVISIONER-STORE]: LOADED from C:\Users\tgruenert\AppData\Roaming\Lens\lens-filesystem-provisioner-store.json +3ms
debug:   ▪ [APP-MAIN] initializing ipc main handlers +3ms
debug:   ▪ [APP-MAIN] Lens protocol routing main +1ms
```
Is there a file location for logging? Or is this all we can see?

**Comment by jishei-128** [comment]
We are experiencing the same issue with 5.5.2. We stick with 5.4.6 for now as a workaround

**Comment by Nokel81** [comment]
Yes, some log files are kept. You can find them https://docs.k8slens.dev/main/faq/?h=logs#where-can-i-find-application-logs

**Comment by tgruenert** [comment]
Found logs and attached one where i tried to connect to a rancher cluster named 'evermind-infra'
[lens1.log](https://github.com/lensapp/lens/files/8830120/lens1.log)

Hope this helped to find a solution. Greetings!

**Comment by mgagliardo91** [comment]
Also seeing the same issue with Rancher and receiving a `Invalid credentials` error message when I can use the same kubeconfig to connect with `kubectl`. Lens is currently unusable in our infrastructure until we can figure this one out, so thanks for prioritizing!

**Comment by Nokel81** [comment]
@mgagliardo91 Your problem will be fixed by #5550

**Comment by Oceanswave** [comment]
Confirmed. Thank you!

**Comment by prowlaiii** [comment]
I'm seeing the error "E0801 17:53:44.809148 18093 proxy_server.go:147] Error while proxying request: getting credentials: exec: executable aws failed with exit code 254" when I'm trying to connect to an AWS EKS cluster.
It's happened with Lens 5.4.6 and 6.0.0.
The Lens kubectl commands work from the command line, using the very same kubeconfig file, eg.

> /Applications/Lens.app/Contents/Resources/x64/kubectl get pods --kubeconfig ~/.kube/myfile --all-namespaces
> 
> NAMESPACE     NAME                       READY   STATUS    RESTARTS   AGE
> 
> kube-system   aws-node-46gn8             1/1     Running   0          5h22m
> 
> kube-system   aws-node-7nbxq             1/1     Running   0          5h22m
> 
> kube-system   aws-node-dvcnm             1/1     Running   0          5h22m
> 
> kube-system   coredns-745979c988-7mrrh   1/1     Running   0          5h28m
> 
> kube-system   coredns-745979c988-nrtmj   1/1     Running   0          5h28m
> 
> kube-system   kube-proxy-b5l7l           1/1     Running   0          5h22m
> 
> kube-system   kube-proxy-chm6q           1/1     Running   0          5h22m
> 
> kube-system   kube-proxy-whf8j           1/1     Running   0          5h22m

Lens lists the clusters in my kubeconfig files (~/.kube/xxxxxxx) but when I click on one of them I just get these errors repeating.

> Starting connection ...
> 
> Authentication proxy started
> 
> Refreshing connection status ...
> 
> An error occurred (InvalidClientTokenId) when calling the AssumeRole operation: The security token included in the request is invalid.
> 
> E0801 18:03:45.798239 18355 proxy_server.go:147] Error while proxying request: getting credentials: exec: executable aws failed with exit code 254
> 
> getting credentials: exec: executable aws failed with exit code 254

I'm on MacOS Monterey 12.5 Intel.

**Comment by MichaelHindley** [comment]
> @mgagliardo91 Your problem will be fixed by #5550

I am still experiencing the "Error while proxying request: Service Unavailable" issue against an AWS EKS cluster and Lens 6, with proxy settings that have worked on 5.4. 

Host is MacOS with cntlm proxy running on 3128 that forwards to internal proxy, with Lens pointing to it in settings.

**Comment by Nokel81** [comment]
Can you open a new issue please
